.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,6 +30,20 @@ eg: Bug fix for widget so that it now correctly thrombs the tubas to provide cle
 - [ ] I have added tests to cover my changes
 - [ ] All new and existing tests passed
 
+## Time frame for completing Pull Request review (from creation date):
+- [ ] As soon as possible
+- [ ] In the next two weeks
+- [ ] In the next month
+
+## Focus of Pull Request:
+<!--- Indicate the main focus of the PR. Additional information can be added below in the Hints section -->
+All comments are welcome regardless of which items are checked off.
+- [ ] All of the below (no need to check off the other boxes)
```

Comment:
```suggestion
- [ ] Inform others of new features and changes
```

I don't the 'all of the below' is necessary; if it is, an 'all of the above' is a more familiar pattern